stabilize.py

```diff
@@ -52,59 +55,52 @@ def calc_movement(flow, criterion="mode"):
                 modeY = st.mode(flow[1])
                 modeZ = st.mode(flow[2])
             except st.StatisticsError:
-                modeX = 0
-                modeY = 0
-                modeZ = 0
+                modeX, modeY, modeZ = 0, 0, 0
             movement = np.array([modeX, modeY, modeZ])
         elif criterion == "median":
             try:
                 medianX = st.median(flow[0])
                 medianY = st.median(flow[1])
                 medianZ = st.median(flow[2])
             except st.StatisticsError:
-                medianX = 0
-                medianY = 0
-                medianZ = 0
+                medianX, medianY, medianZ = 0, 0, 0
             movement = np.array([medianX, medianY, medianZ])
         elif criterion == "mean":
             try:
-                meanX = sum(flow[0])/len(flow[0])
-                meanY = sum(flow[1])/len(flow[1])
-                meanZ = sum(flow[2])/len(flow[2])
+                meanX, meanY, meanZ = np.mean(flow,axis=1)
             except ZeroDivisionError:
-                meanX = 0
-                meanY = 0
-                meanZ = 0
+                meanX, meanY, meanZ = 0, 0, 0
             movement = np.array([meanX, meanY, meanZ])
+        else:
+            print("Error: criterion")
```

Comment:
"BAD criterion: {}.  Select from \"mode\", \"median\", and \"mean\".".format(criterion)